database.py

```diff
@@ -72,7 +72,7 @@ def __utcnow_between(self, start:datetime.datetime, end:datetime.datetime):  #tr
     def __utctoday_or_between_not_past(self, start:datetime.datetime, end:datetime.datetime): #true if today or between
         return (self.__is_utctoday(start) or self.__is_utctoday(end) or self.__utcnow_between(start,end)) and self.__not_in_past(end)
 
-    def __announce(self, daily_time= datetime.time(13,0), pause = datetime.timedelta(hours=6)):   #true if pause hours before daily_time or after daily
+    def __announce(self, daily_time= datetime.time(11,0), pause = datetime.timedelta(hours=6)):   #true if pause hours before daily_time or after daily
```

Comment:
Passt das jetzt schon? Wofür stehen noch diese 6 Stunden Timedelta?
Du kannst auch einfach mal alles auf 12:15 abändern und dann können wir im Testing nochmal sehen ob es klappt.

Reply:
Bin gerade dabei dieses Datum in eine variable um zu bauen
Die 6 Stunden sind ein Buffer in welchem keine Nachrichten kommen falls sie in dem Zeitraum vor dem Daily passieren
Das ist damit das daily noch relevanz hat, sonst hätte man ja schon alle infos